### PR TITLE
Horizontal swing, quiet mode, hvac mode support and bugfixes

### LINKED ISSUE #3: Allow to set horizontal swing
Possible values: `default, full, fixedLeft, fixedMidLeft, fixedMid, fixedMidRight, fixedRight`

### PR DESCRIPTION
**Pull request by denniedegroot**
This fixes #3 #8 #63 #95 #109

For Homey Pro 2023 I introduced hvac_mode next to thermostat_mode but they are synced.

### CODE REVIEW

**Comment by aivus** [comment]
Hello @denniedegroot 

Sorry for such long delay with response. I've cherry-picked(https://github.com/aivus/com.gree/pull/129) you changes to get rid of non-finished #95.

Will release it to Test channel as 0.8.0

File: app.js [hunk 1]
```diff
@@ -53,11 +53,25 @@ class GreeHVAC extends Homey.App {
...
+                const horizontalSwing = args.device.getCapabilityValue('horizontal_swing');
+                args.device.log('[condition]', '[current swing horizontal]', horizontalSwing);
+                return args.horizontal_swing === horizontalSwing;
+            });
+
+        this._conditionQuietModeIs = this.homey.flow.getConditionCard('quiet_mode_is')
+            .registerRunListener((args, state) => {
+                const horizontalSwing = args.device.getCapabilityValue('quiet_mode_is');
+                args.device.log('[condition]', '[quiet mode]', horizontalSwing);
+                return args.horizontal_swing === horizontalSwing;
```
Comment (aivus) [inline app.js:67]: looks like should be `args.mode`

Comment (denniedegroot) [reply app.js:67]: Yes this is correct.

**Comment by denniedegroot** [comment]
When will there be a test version released?

**Comment by aivus** [comment]
Published as [0.8.0 to test](https://homey.app/de-de/app/com.gree/Gree/test/)

**Pull request closed without merging**